### ISSUE TITLE
suggestion for dragons in the 1.16 nether

### ISSUE THREAD

**Issue by ShadowDMA** [AlexModGuy/Ice-and-Fire-Suggestions#694]
Maybe there could be fire dragon variants in the nether like one the breaths the new blue flames in 1.16 or one that has a texture like magma blocks. Maybe there is a super rare chance for a mad lad ice dragon living in the nether lol

**Comment by kodabeer** [comment]
they've stated there won't be any new dragons other than lightning dragon

**Comment by ShadowDMA** [comment]
I assumed they said that in regard to adding new dragon types other than fire, ice, lightning, like the people that ask for crazy things like earth dragons light dragons ender dragons etc.
This is basically still a fire dragon just spawning in the nether with appropiate textures, they also said that long before we knew anything about a nether update

**Comment by Xorbah** [comment]
No, they said that as in no new types whatsoever. This would also apply to
variants.

**Comment by ShadowDMA** [comment]
Well as I said this is an extenuating circumstance in that the nether has changed from being a barren wasteland to an interesting location I am not asking for some stupid elemental dragon just putting it out there that it might be worth reconsidering dragons in the nether due to the circumstances and hoping someone would discuss the idea constructively
